app/workers/queue_listener.rb

```diff
@@ -6,22 +6,18 @@ class QueueListener
   # shoryuken_options queue: 'post_test', auto_delete: true, body_parser: :json
 
   def perform(sqs_message, params)
-    params = params.symbolize_keys
```

Comment:
Interesting; I'm not sure why I put those conversions in there, then. Gut says it was a testing thing, but if tests are passing, I've got no reservations.